README.md

```diff
@@ -106,7 +115,7 @@ To update the state of your currently deployed integrations, hover over the **Ap
 
 ![Apache Camel K Integrations view - Refresh](images/camelk-integrations-view-refresh-action.jpg)
 
-## Creating a new Camel K Integration with multiple parameters
+## Creating a new Camel K Integration task configuration with multiple parameters
```

Comment:
I changed it in this PR as I was finding that it was ambiguous with the "new Camel K Integration file" added feature